core/src/main/java/nl/nn/adapterframework/receivers/ExchangeMailListener.java

```diff
@@ -329,12 +344,21 @@ public boolean isSimple() {
 		return simple;
 	}
 
+	@Deprecated
+	@ConfigurationWarning("use storeResultInSessionKey instead")
 	public void setStoreEmailAsStreamInSessionKey(String string) {
 		storeEmailAsStreamInSessionKey = string;
 	}
 	public String getStoreEmailAsStreamInSessionKey() {
 		return storeEmailAsStreamInSessionKey;
 	}
 
+	public void setStoreEmailInSessionKey(String string) {
+		storeEmailInSessionKey = string;
+	}
+	public String getStoreEmailInSessionKey() {
+		return storeEmailInSessionKey;
+	}
```

Comment:
Deze functionaliteit is helemaal niet nodig. Als je meteen al over de email wil beschikken, dan kan je gewoon messageType op 'contents' zetten. Dat moet dan ook in de deprecated-warnings staan, en deze nieuwe optie moet weg.